base_user_role/models/role.py

```diff
@@ -75,6 +105,30 @@ def cron_update_users(self):
         logging.info("Update user roles")
         self.search([]).update_users()
 
+    def show_rule_ids(self):
+        return {
+            "name": "Record Rules",
+            "type": "ir.actions.act_window",
+            "res_id": self.id,
+            "view_mode": "tree",
+            "res_model": "ir.rule",
+            "view_id": self.env.ref("base.view_rule_tree").id,
+            "target": "current",
+            "domain": [("id", "in", self.rule_ids.ids)],
+        }
```

Comment:
Could we reuse the existing action and tune it instead (we avoid to translate new terms this way)?
```suggestion
        action = self.env["ir.actions.actions"]._for_xml_id("base.action_rule")
        action["domain"] = [("id", "in", self.rule_ids.ids)]
        return action
```